.github/workflows/release.yml

```diff
@@ -0,0 +1,133 @@
+name: Release
+
+on:
+  # pull_request:
+    # branches: [ master ]
+  push:
+    tags:
+      - "*beta*"
+      - "*pre*"
+      # - "1[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  prerelease:
+    name: Pre-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout highlight.js
+        uses: actions/checkout@v2
+
+      - name: Tag is ${{ github.ref }}.
+        # we have to repeat ourselves here since the environment is not actually updated
+        # until the next step executes, so we can't access $TAG_NAME yet
+        run: |
+          echo "TAG_NAME=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+          echo "MAJOR_VERSION=$(echo ${GITHUB_REF/refs\/tags\//} | cut -d'.' -f1)" >> $GITHUB_ENV
+      - name: Make sure we are pushing a tag...
+        if: ${{ !contains(github.ref,'refs/tags/') }}
```

Comment:
```suggestion
        if: !contains(github.ref,'refs/tags/')
```

Reply:
It doesn’t like this and my editor color is it funny I think that some type of weird YAML thing.

Reply:
Or is the trick that you surround the whole expression in parentheses I’m not sure I tried that

Reply:
I think it may just be some weird highlighting with editors, according to GitHub's spec...

> When you use expressions in an if conditional, you may omit the expression syntax (`${{ }}`) because GitHub automatically evaluates the if conditional as an expression.
>
> Source: https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions

Reply:
Yes you told me that and sometimes it works but yet most of the examples I see even on GitHub’s on site I have the brackets all the time.

Reply:
I spent way too long on this one so I’m inclined to leave it alone for now since it works.